src/components/forms/DeploymentForm.vue

```diff
@@ -0,0 +1,213 @@
+<template>
+  <BaseForm @submit.native.prevent="handleSubmit" class="form">
+    <header>
+      <h2 class="title">
+        Deploy #{{buildNumber}}
```

Comment:
we should remove the term "deploy" since this tends to confuse people.  This is why we change the term "deploy" to "promote" in 1.0 so that it was less confusing

Reply:
This form is being used for rollback as well, do I dynamically change the title based on the `promote/rollback` event?

Reply:
yes, that sounds like a good idea, for example

> Promote #42 

and

> Rollback to #42

Reply:
actually, you may be able to remove the build number completely from the popup.  Presumably a user knows which repository and build they are promoting, and the build number should also be in url bar and title.

Reply:
@bradrydzewski what about the button label, should I name it "Environments"